src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java

```diff
@@ -60,14 +63,24 @@ private CallbackResponse makePaymentServiceReq(CallbackParams callbackParams) {
             log.info("calling payment service request {}", caseData.getCcdCaseReference());
             var serviceRequestReference = paymentsService.createServiceRequest(caseData, authToken)
                 .getServiceRequestReference();
+            log.info("Service Request Reference {}", serviceRequestReference);
+            if (caseData.getHearingDueDate() != null) {
+                try {
+                    Fee hearingFeeChecked = feesService.getHearingFeeDataByTotalClaimAmount(
+                        caseData.getClaimValue().getStatementOfValueInPennies());
+                    log.info("Hearing Fee Checked {} {} {}", hearingFeeChecked.getCalculatedAmountInPence().toString(),
+                             hearingFeeChecked.getCode(), hearingFeeChecked.getVersion());
+                } catch (Exception e) {
+                    log.info("Hearing Fee check failed");
+                }
```

Comment:
When I see try/catch it always sets me off. It might be perfectly fine, but I need to ask, especially if there is no dedicated comment.

- Why is it expected that the call to getHearingFeeDataByTotalClaimAmount could throw an exception?
- What are the implications of a failure in calling getHearingFeeDataByTotalClaimAmount?

Reply:
it was only a check for testing, I have removed it

Reply:
Ah, ok.